kde-misc/plasma-applet-active-window-control/plasma-applet-active-window-control-1.7.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="Plasma 5 applet for controlling currently active window"
+HOMEPAGE="https://store.kde.org/p/998910/ \
+	https://github.com/kotelnik/plasma-applet-active-window-control"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/kotelnik/${PN}.git"
+else
+	SRC_URI="https://github.com/kotelnik/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+IUSE=""
+LICENSE="GPL-2+"
+SLOT="5"
+
+RDEPEND="
+	$(add_frameworks_dep kconfig)
+	$(add_frameworks_dep kcoreaddons)
+	$(add_frameworks_dep plasma X)
+	$(add_qt_dep qtcore)
```

Comment:
drop it with kde5.eclass

Reply:
All of them?

Reply:
only qtcore, because kde5.eclass already depends on qtcore ;)